### DISCUSSION TITLE
How do I report an issue with documentation?

### DISCUSSION THREAD

**Discussion by Nick-Minutello** [remix-run/remix#9795] [Q&A] [answered]
Nothing massive - the code example is actually a bit wrong (the `revalidator` in the `useEffect` dependency will result in it polling without any delay - making the interval pointless).

Also the `useInterval(5000)` is something that exists? If it is, If so, I haven't been able to find it.

https://remix.run/docs/en/main/hooks/use-revalidator#revalidatorrevalidate

**Comment by sergiodxa** [comment, marked as answer]
The docs are here in this same repository, you can send a PR to fix any issue.

The useInterval hook is there as an example, it's not provided by Remix.

**Reply by Nick-Minutello** [reply]
Ok. Thanks.
Does that particular `useInterval` hook actually exist? Or is it just hypothetical code?
(I found a number of `useInterval` implementations, but the result was always quite a bit more complicated/difficult than the code in this example.

**Reply by sergiodxa** [reply]
It probably exists but in the context of the docs is hypothetical code.

**Reply by ericraymond** [reply]
Wow.  Example code that does not run  (and does not have reasonable includes) is less than useful.
